special-pages/pages/new-tab/app/privacy-stats/integration-tests/privacy-stats.page.js

```diff
@@ -103,8 +103,8 @@ export class PrivacyStatsPage {
         // open the NTP with the correct param
         await this.ntp.openPage({ additional: { stats: 'fewOnlyTop' } });
 
-        // control, ensure we have 5 rows
-        expect(await rows.count()).toBe(3);
+        // control, ensure we have 3 rows
+        await expect(rows).toHaveCount(3);
```

Comment:
switching to macos runner for the playwright tests changed the perf characteristics of tests. So these changes use auto-waiting variants in playwright to make them less flaky.